### DISCUSSION TITLE
Does Laravel11 comes with doctrine/dbal pre-installed?

### DISCUSSION THREAD

**Discussion by yogeshgalav** [laravel/framework#50779] [Q&A] [answered]
if not please suggest the solution to dependency issue
        "doctrine/dbal": "^2.13.9",

![image](https://github.com/laravel/framework/assets/26108890/20d7a8b5-dd7c-469c-945b-55101f0f7ec7)

**Comment by crynobone** [comment]
Laravel Framework 11 doesn't rely on doctrine/dbal. It could be possible your other dependencies still rely on it and prevent the installation.

**Reply by yogeshgalav** [reply]
I upgraded my Laravel application to version11, 
earlier version had dbal installed for altering the tables migration,
When installing dbal on new version 11, it gave error
`require symfony/http-foundation`

That's core requirement right how to resolve that?

**Reply by crynobone** [reply, marked as answer]
https://laravel.com/docs/11.x/upgrade#doctrine-dbal-removal
